src/textual/widgets/_tree.py

```diff
@@ -334,43 +334,67 @@ class NodeMessage(Generic[EventTreeDataType], Message, bubble=True):
         """Base class for events sent when something happens with a node.
 
         Attributes:
-            TreeNode[EventTreeDataType]: The node involved in the event.
+            node: The node involved in the event.
```

Comment:
I think we can discard this message altogether. We don't use it as a base class now, and we never explicitly send it.

Reply:
I don't know why I kept it 🤷 
Removed o7